pulp_ci_centos/Containerfile

```diff
@@ -2,10 +2,15 @@ FROM quay.io/centos/centos:stream8
 # https://ryandaniels.ca/blog/docker-dockerfile-arg-from-arg-trouble/
 ARG SCHEME="http"
 
-COPY s6-overlay-amd64.tar.gz /tmp
+COPY s6-overlay-x86_64.tar.gz /tmp
+COPY s6-overlay-noarch.tar.gz /tmp
+COPY s6-overlay-symlinks-noarch.tar.gz /tmp
+COPY s6-overlay-symlinks-arch.tar.gz /tmp
 
-RUN tar xzf /tmp/s6-overlay-amd64.tar.gz -C / --exclude="./bin" && \
-    tar xzf /tmp/s6-overlay-amd64.tar.gz -C /usr ./bin
+RUN tar xzf /tmp/s6-overlay-x86_64.tar.gz -C /
+RUN tar xvf /tmp/s6-overlay-noarch.tar.gz -C /
+RUN tar xvf /tmp/s6-overlay-symlinks-noarch.tar.gz -C /
+RUN tar xvf /tmp/s6-overlay-symlinks-arch.tar.gz -C /
```

Comment:
Would we save more space by removing those files again?

Reply:
I think it depends on whether layers are in use or not. Either way, I'll try removing them immediately afterwards.